documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2024,6 +2024,9 @@ In order to capture large object values and have them serialized in the change e
 |
 |A comma-separated list of RAC node host names or addresses.
 This field is required to enable Oracle RAC support.
+If a RAC node uses a port that differs from `database.port`, the entry can be specified as `ip:port` to provide the node's custom port. +
```

Comment:
Hi @roldanbob, it's pronounced _/rak/_.

Reply:
> On first use of "RAC" (which appears to be in the description of the `database.url` property), expand to _Oracle Real Application Clusters (RAC)_. I think that I did that in the PR for DBZ-3407.

If you took care of that in the other PR, I'll leave it as-is here to avoid any type of merge conflicts.

Reply:
> it's pronounced _/rak/_.

Ok, so no change needed wrt to the indefinite article.

Reply:
>If you took care of that in the other PR, I'll leave it as-is here to avoid any type of merge conflicts.

As for the expanded acronym, in DBZ-3407 it looks like I expand it on L2241, but I missed first use on L1874. A nice-to-fix, but not a huge deal, so I don't think any action is necessarily required.